tutorials/krusty/with_sockeye/sockeye.i

```diff
@@ -0,0 +1,441 @@
+################################################################################
+## KRUSTY                                                                     ##
+## Heat Pipe Microreactor Steady State                                        ##
+##                                                                            ##
+## Mahmoud (3/25) (openmc.i/solid.i/sockeye.i/model03.py/krusty_scale.e)      ##
+################################################################################
+
+# Total heat removed/added to heat pipe
+Q_hp = 500.0
+
+# Wick characteristics
+R_pore = 15.0e-6
+D_h_pore = '${fparse 2.0 * R_pore}'
+permeability = 2e-9
+porosity = 0.70
+
+# Envelope ("env")
+# SS316. Incropera & DeWitt, 3rd ed, Table A.1 @ 900K (627C)
+# Density (kg/m3)
+rho_env = 8238.
+# Thermal conductivity (W/m-K)
+k_env = 23.05
+# Specific heat capacity (J/kg-K)
+cp_env = 589.
+
+# Sodium vapor
+# From Appendix C of text book "Heat Pipe Design and Technology"
+# Sat. Vapor Sodium at T = 800C (1073.15 K)
+# Density (kg/m3)
+rho_vapor = 0.134
+# Effective "super" conductivity (W/m-K)
+#k_vapor = 1.0e5 # Reaches 600 seconds
+k_vapor = 1.0e6
+# Specific heat capacity (J/kg-K)
+cp_vapor = 9040.
+
+# Sodium liquid
+# From Appendix C of text book "Heat Pipe Design and Technology"
+# Sat. Liquid Potassium at T = 800C (1073.15 K)
+# Density (kg/m3)
+rho_liquid = 757.3
+# Thermal conductivity (W/m-K)
+k_liquid = 57.81
+# Specific heat capacity (J/kg-K)
+# From Table 1.1, no temperature data given
+cp_liquid = 1380.
+# Melting point, Table 3.1, lists 98C, boiling point at atm. 892C
+
+# Wick, homogenize envelope and fluid
+# Density (kg/m3)
+rho_wick = '${fparse porosity * rho_liquid + (1.0 - porosity) * rho_env}'
+# Thermal conductivity (W/m-K)
+k_wick = '${fparse porosity * k_liquid + (1.0 - porosity) * k_env}'
+# Specific heat capacity (J/kg-K)
+# From Table 1.1, no temperature data given
+cp_wick = '${fparse porosity * cp_liquid + (1.0 - porosity) * cp_env}'
+
+# Elevations and lengths
+# Note: For blackbox model -- manually update "length" input
+length_evap = 68.4e-2
+length_adia = 11.4e-2
+length_cond = 34.2e-2
+
+# Mesh density
+# The dimensions are nicely divisible by 3 cm mesh.
+nelem_base_evap = 50
+nelem_base_adia = 10
+nelem_base_cond = 30
+mesh_density = 3
+nelem_evap = '${fparse mesh_density*nelem_base_evap}'
+nelem_adia = '${fparse mesh_density*nelem_base_adia}'
+nelem_cond = '${fparse mesh_density*nelem_base_cond}'
+
+# Envelope thickness
+t_env = 0.089e-2
+# Liquid annulus thickness
+t_ann = 0.07e-2
+# Wick thickness
+t_wick = 0.1e-2
+
+# Radial geometry
+# Envelope outer
+R_hp_o = 1.27e-2
+D_hp_o = '${fparse 2.0 * R_hp_o}'
```

Comment:
As @MendesMichael pointed out earlier, heat pipe outer diameter should be 1.27e-2 cm